packages/core/database/lib/dialects/dialect.js

```diff
@@ -50,6 +48,10 @@ class Dialect {
   canAddIncrements() {
     return true;
   }
+
+  supportsWindowRowNumber() {
```

Comment:
I didn't dig into if this is a general supportsWindowFunctions or not @Marc-Roig do you know if this can be more general ?

Reply:
I am looking at it 👀 . Also, super minor thing, could we group the supportsX & canAdd functions one after the other?
There is one  supportsUnsigned and canAlterConstraints above.

Reply:
Just pushed :)

Reply:
In mysql 8 they introduced Window functions, which is not only row number, but also the ones listed [here](https://arctype.com/blog/mysql-window-functions/) (CUME_DIST, DENSE_RANK, RANK, ...)

So maybe we could rename this to `supportsWindowFunctions()`

Reply:
Let's do that then 👍